### PR TITLE
Dera development

### PR DESCRIPTION
**Pull request by derachimso**

### CODE REVIEW

File: capstone/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,30 +2,30 @@
...
-        <div class="welcome">
-            <h1 class="cover-heading">Welcome.</h1>
-            <p class="lead">
-                This website will give you election Information
-                relevant to you, and allow you add election dates to your 
-                Google Calendar.
-            </p>
-        </div>
+    <div class="welcome">
+        <h1 class="cover-heading">Welcome.</h1>
```
Comment (magwitz) [inline capstone/src/main/webapp/index.html:12]: If you look at the linter, it specifies lines with errors. Line 12 is one of them.
It's indented 4 spaces more than the line above, when it should only be 1-3 spaces. (One way to fix it is just to make it even with the line above, then hit tab)

File: capstone/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,30 +2,30 @@
...
-                <input type="address" id="address" name="address" class="form-control" placeholder="Enter Address" required autofocus>
-                <input type="electionID" id="inputID" name="inputID" class="form-control" placeholder="Election ID" required>
-                <input type="submit" class="btn btn-lg btn-primary btn-block">
-            </form>
-        </div>
+    <br>
+      
+    <div class="inputinfo">
+        <form class="form-input" action="/VoterServlet1">
+            <label for="Title" class="sr-only">Enter Your Information Here:</label>
```
Comment (magwitz) [inline capstone/src/main/webapp/index.html:24]: You don't need "for='Title'" here
The attribute "for" binds the label to a form element & in this case, it's not binding it to anything

File: capstone/src/main/java/com/google/sps/servlets/VoterServlet.java [hunk 3]
```diff
@@ -13,22 +15,31 @@
...
-    //         "address=%s&inputID=%s",
-    //         URLEncoder.encode(address, charset), URLEncoder.encode(inputID, charset));
-    // URLConnection connection = new URL(url + "?" + query).openConnection();
-    // connection.setRequestProperty("Accept-Charset", charset);
-    // InputStream result = connection.getInputStream();
+    String electionId = request.getParameter("electionId");
+    String address = request.getParameter("address");
+    Dotenv dotenv = Dotenv.configure().directory(System.getProperty("user.dir")).load();
+    String key_prod = dotenv.get("API_KEY");
+    String Key_Test = "";
```
Comment (magwitz) [inline capstone/src/main/java/com/google/sps/servlets/VoterServlet.java:22]: This variable never gets set. Do we need it for this to run? If not, delete this and the logic below that sets it to key_prod & just use key_prod.

File: capstone/src/main/webapp/WEB-INF/code.java [hunk 4]
```diff
@@ -0,0 +1,10 @@
+public class Event{
```
Comment (magwitz) [inline capstone/src/main/webapp/WEB-INF/code.java:1]: What is this file for?

File: capstone/src/main/java/com/google/sps/servlets/VoterServlet.java (hunk 3 above)
Comment (derachimso) [reply capstone/src/main/java/com/google/sps/servlets/VoterServlet.java:22]: Key prod is supposed to have restrictions that prevent us from deploying the code locally which is why we have key test for local testing. Before I push to github I take the test API out so it uses key prod instead. I don't know if it makes sense

File: capstone/src/main/webapp/WEB-INF/code.java (hunk 4 above)
Comment (derachimso) [reply capstone/src/main/webapp/WEB-INF/code.java:1]: deleted.

File: capstone/src/main/java/com/google/sps/servlets/VoterServlet.java (hunk 3 above)
Comment (magwitz) [reply capstone/src/main/java/com/google/sps/servlets/VoterServlet.java:22]: I think there might still be some confusion here, but we can wait for Chen to weigh in and in the meantime, you can go ahead and submit.

**Pull request merged**